mappings/net/minecraft/util/registry/RegistryKey.mapping

```diff
@@ -1,16 +1,45 @@
 CLASS net/minecraft/class_5321 net/minecraft/util/registry/RegistryKey
+	COMMENT Represents a key for a value in a registry in a static context.
```

Comment:
I'm not sure what "static context" means here. Maybe clarify a bit more with a longer sentence?